### PR TITLE
Keycloak upgrade

### PR DESCRIPTION
**Pull request by alanraju-aot**
changes to support keycloak 23 upgrade

### CODE REVIEW

File: docker-compose/docker-compose-arm64.yml [hunk 1]
```diff
@@ -2,32 +2,40 @@ version: '3.7'
...
       POSTGRES_PASSWORD: ${KEYCLOAK_JDBC_PASSWORD:-changeme}
-      PGDATA: /var/lib/postgresql/data
+    ports:
+      - 5431:5431
     networks:
       - formsflow
 
   keycloak:
-    image: formsflow/forms-flow-keycloak-arm64:14.0.0
+    image: quay.io/keycloak/keycloak:21.1.1
```
Comment (sumesh-aot) [inline docker-compose/docker-compose-arm64.yml:20]: Can you change it to keycloak 23 ?

File: docker-compose/docker-compose-ee.yml [hunk 2]
```diff
@@ -2,32 +2,40 @@ version: '3.7'
...
       POSTGRES_PASSWORD: ${KEYCLOAK_JDBC_PASSWORD:-changeme}
-      PGDATA: /var/lib/postgresql/data
+    ports:
+      - 5431:5431
     networks:
       - formsflow
 
   keycloak:
-    image: quay.io/keycloak/keycloak:14.0.0
+    image: quay.io/keycloak/keycloak:21.1.1
```
Comment (sumesh-aot) [inline docker-compose/docker-compose-ee.yml:20]: Same here

File: docker-compose/docker-compose.yml [hunk 3]
```diff
@@ -2,32 +2,41 @@ version: '3.7'
...
       POSTGRES_PASSWORD: ${KEYCLOAK_JDBC_PASSWORD:-changeme}
-      PGDATA: /var/lib/postgresql/data
+    ports:
+      - 5431:5431
     networks:
       - formsflow
 
   keycloak:
-    image: quay.io/keycloak/keycloak:14.0.0
+    image: quay.io/keycloak/keycloak:21.1.1
```
Comment (sumesh-aot) [inline docker-compose/docker-compose.yml:20]: Here too

File: docker-compose/docker-compose-arm64.yml (hunk 1 above)
Comment (alanraju-aot) [reply docker-compose/docker-compose-arm64.yml:20]: yes sure

**Review by sumesh-aot — APPROVED** [review]

File: docker-compose/docker-compose.yml [hunk 4]
```diff
@@ -2,32 +2,41 @@ version: '3.7'
...
-      PGDATA: /var/lib/postgresql/data
+    ports:
+      - 5431:5431
     networks:
       - formsflow
 
   keycloak:
-    image: quay.io/keycloak/keycloak:14.0.0
+    image: quay.io/keycloak/keycloak:23.0.1
+    restart: always
```
Comment (auslin-aot) [inline docker-compose/docker-compose.yml:21]: keycloak version should be 23.0.7?

Comment (alanraju-aot) [reply docker-compose/docker-compose.yml:21]: done

**Review by auslin-aot — APPROVED** [review]

**Pull request merged**